redfish-core/include/query.hpp

```diff
@@ -27,7 +27,9 @@
 // IWYU pragma: no_include <boost/url/impl/params_view.hpp>
 // IWYU pragma: no_include <boost/url/impl/url_view.hpp>
 
+#ifdef BMCWEB_ENABLE_REDFISH_AGGREGATION
```

Comment:
We stopped doing this for the includes

Reply:
otherwise this is pulling aggregator code and compiling